Resources/Prototypes/_EE/Gamerules/SANDeserters.yml

```
@@ -0,0 +1,77 @@
+    minimumPlayers: 20
+    duration: 1
+    maxOccurrences: 1
+  - type: LoadMapRule
+    preloadedGrid: SANDesertersDropship
+
+- type: preloadedGrid
+  id: SANDesertersDropship
+  path: /Maps/Shuttles/ShuttleEvent/SANDropship.yml
+  copies: 1
+
+- type: entity # Part of PirateRadioSpawn
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)